src/main-page.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-cycle */
+import { createNavigation, addAllEventListeners } from './nav-creator';
+/* eslint-enable import/no-cycle */
+
+function createMainPage() {
```

Comment:
- [ ] [required] You should use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
  - [ ] [required] Usage of arrow functions instead of the `function` keyword